### ISSUE TITLE
Long line numbers pushes favorite-icon out of view

### ISSUE THREAD

**Issue by hildeor** [AtB-AS/mittatb-app#983]
## Origin

_Links to received feedback, user research or other findings._

Feedback from user in Intercom, where line 16 is impossible to mark as favorite due to the line name.
This is also a problem that occurs when text size is increased.

![favorite](https://user-images.githubusercontent.com/8980523/110463145-d4b3b780-80d1-11eb-8c6a-47c96fd84d27.png)

## Motivation

_A short description of what user needs or business goals this feature will solve._

## Hypotheses and assumptions

_A list of hypotheses and assumptions we have made about the user or the proposed solution._

## Proposed solution

_A coarse description of a proposed solution, that may include wireframes and graphic design._

- Favorite icons position should be persistent, regardless of length on line name and text size
- Break line number and name over several lines if necessary

## Acceptance Criteria

_List of relevant acceptance criteria as a part of a QA flow_

- [] Renders colors as expected in dark and light mode.
- [] Is translatable to English and Norwegian.
- [] Does not drain battery
- [] Works in up to 200% font size
- [] Is operable and perceivable using screen reader
